src/test/java/org/gridsuite/study/server/utils/WireMockUtils.java

```diff
@@ -423,4 +422,15 @@ public void verifyFilterExport(UUID stubUuid, String filterUuid, String networkU
         verifyGetRequest(stubUuid, "/v1/filters/" + filterUuid + "/export",
                 Map.of(NETWORK_UUID, WireMock.equalTo(networkUuid)));
     }
+
+    public UUID stubGetProfileId(String userId) {
+        return wireMock.stubFor(WireMock.get(WireMock.urlPathEqualTo("/v1/users/" + userId + "/profile"))
+                // return valid user data without params
+                .willReturn(WireMock.okJson("{\"id\":\"97bb1890-a90c-43c3-a004-e631246d42d6\",\"name\":\"Profile No params\"}")))
+                .getId();
+    }
+
+    public void verifyGetProfileId(UUID stubUuid, String userId) {
+        verifyGetRequest(stubUuid, "/v1/users/" + userId + "/profile", Map.of());
+    }
```

Comment:
Is that necessary ?

Reply:
Yes, that's used in StudyServiceTest class 6 times each.